src/content-handlers/iiif/modules/uv-contentleftpanel-module/ContentLeftPanel.ts

```diff
@@ -491,26 +491,28 @@ export class ContentLeftPanel extends LeftPanel<ContentLeftPanelConfig> {
     }
 
     const paged: boolean =
-      !!this.extension.getSettings().pagingEnabled &&
-      this.extension.helper.isPaged();
-
-    const selectedIndices: number[] = this.extension.getPagedIndices(
-      this.extension.helper.canvasIndex
-    );
-
-    // console.log("selectedIndeces", selectedIndices);
-
-    this.thumbsRoot.render(
-      createElement(ThumbsView, {
-        thumbs,
-        paged,
-        viewingDirection: viewingDirection || ViewingDirection.LEFT_TO_RIGHT,
-        selected: selectedIndices,
-        onClick: (thumb: Thumb) => {
-          this.extensionHost.publish(IIIFEvents.THUMB_SELECTED, thumb);
-        },
-      })
-    );
+    !!this.extension.getSettings().pagingEnabled &&
+    this.extension.helper.isPaged();
+  
+  const selectedIndices: number[] = this.extension.getPagedIndices(
+    this.extension.helper.canvasIndex
+  );
+  
+  const settings = this.extension.getSettings(); 
+  
+  this.thumbsRoot.render(
+    createElement(ThumbsView, {
+      thumbs,
+      paged,
+      viewingDirection: viewingDirection || ViewingDirection.LEFT_TO_RIGHT,
+      selected: selectedIndices,
+      truncateThumbnailLabels: settings.truncateThumbnailLabels || false,
```

Comment:
I think @edsilv had suggested defaulting this to true rather than false. I don't personally have a strong preference, though, so if you feel otherwise, we can discuss further. :-)

Reply:
When I change it to true the checkbox stops working and I'm not sure why - I've tried changing the css around and making this part "settings.truncateThumbnailLabels = this.$truncateThumbnailLabelsCheckbox.is(":checked");" false but neither worked

Reply:
Is the problem that when you build the checkbox in the SettingsDialogue, it is always being initialized to unchecked? Does that code need to look at the settings in order to start out in the correct state?

Reply:
I got it working by changing it so that it's now checked by default and unchecking it truncates the label, and I changed the wording in the settings dialogue so it still makes sense. I don't think either way is necessarily better, just down to preference?

Reply:
I don't have strong feelings about one way being better than the other -- but if this changes the default behavior of the UV, it's a more disruptive change than if it maintained the status quo but offered the option to change things. If this PR makes visible changes to everyone's UV by default, it might cause more controversy than if it didn't... but if people generally prefer the full labels, maybe it's for the best. (For my part, I wouldn't mind the change... just not sure what others might think).

Have you been able to test whether overriding the default behavior via a JSON file works correctly here? I think as long as the configuration file gives us control over it, we're in a pretty good position regardless of which default we choose. But if the override isn't working yet, we'll want to address that prior to merging.

Reply:
Previously I had it so that the labels were still truncated by default and checking the box shows the overflow, but changed it to how it is now after the community call. The difference between today's commit and the previous one is that for this one, the box is checked by default and unchecking it truncates the label, and it defaults to true, while the last one has the box unchecked by default and checking it truncates the label instead, and it defaults to false.  
I have tested it, I'll update the config file

Reply:
Thanks, @Saira-A -- I'll be sure this gets discussed on next week's Steering Committee call to see if we can get a consensus about moving it forward.